alphagov/paas-cf
# [#154760148] Unify post-deploy tasks

## What

At the moment, we have two places where some post-deploy tasks are run.
One is sort of inline with the cf-deploy and the other has a dedicated
separate task. We think this will get confusing as to where any future
tasks would come in.

Here, we're moving them back into cf-deploy as it requires less
refactoring to achieve what we need, which is running the tests in the
same time as cf-deploy and post-deploy tasks.

## How to review

- Run the pipelines and expect the `cf-deploy` task to succeed
- Make sure the tasks moved have been executed
- Make sure the tasks moved are in the right place

## Who can review

I'd like it to be someone from the initial pair that implemented a new task. /cc @chrisfarms @henrytk 

But I imagine anyone could pick that up.
